code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -470,6 +462,19 @@
 	else
 		return FALSE
 
+/obj/structure/closet/wrench_act_secondary(mob/living/user, obj/item/tool)
+	if(opened && anchorable) // The locker bolts are on the inside, so we make sure the locker is open before we mess with them
+		if(isinspace() && !anchored)
+			balloon_alert(user, "nothing to anchor to!")
+			return TRUE
+		set_anchored(!anchored)
+		tool.play_tool_sound(src, 75)
+		user.balloon_alert_to_viewers("[anchored ? "anchored" : "unanchored"]")
+		return TRUE
+	if(!opened) // The locker is closed, so the bolts aren't accessible
+		balloon_alert(user, "locker must be open!")
+		return TRUE
```

Comment:
Instead of having `if(opened)` and `if(!opened)` checks you could do `if` and `else`.
You could also make usage of early returns. For example you could start with
```dm
if(!opened) // The locker is closed, so the bolts aren't accessible
		balloon_alert(user, "locker must be open!")
		return TRUE
```
Then you can simply check `if(anchorable)`, as you'll know that from that point onwards in the code the locker is guaranteed to be open (else it'd have returned).

Reply:
That makes sense. Is this just for my future reference, or would you like me to make these changes?

Reply:
If you can make them, that'd be good, yea.